packages/gatsby-remark-prismjs/README.md

```diff
@@ -228,6 +228,46 @@ You can also add line highlighting. It adds a span around lines of code with a
 special class `.gatsby-highlight-code-line` that you can target with styles. See
 this README for more info.
 
+To highlight lines, you can use one of the following directives as comments in your
+code:
+
+- `highlight-line` highlights the current line;
+- `highlight-next-line` highlights the next line;
+- `highlight-start` highlights the lines until the matching `hightlight-end`;
+- `highlight-range{1, 4-6}` will highlight the next line, and the fourth, fifth and sixth lines.
+
+  ```javascript
+  function calculateWinner(squares) {
+    //highlight-line
```

Comment:
What would this actually do? Probably want to clean up this example a tad.

Reply:
Gave it another spin in 3ebbdbd, should make it more clear/explicit